nemo_aligner/data/nlp/scripts/undo_special_tokens.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to remove special tokens from dpo datasets 
+and convert them into list of messages format"""
+
+import json
+import re
+import sys
+
+input_jsonl = sys.argv[1]
```

Comment:
could we write this in the:

```python
if __name__ == '__main__':
```

pattern? It helps when someone needs to import modules to test. If you refactor, can you also write a unit test to make sure this function does what you expect so future folks don't break the desired behavior?